can-define.js

```diff
@@ -745,7 +745,7 @@ getDefinitionsAndMethods = function(defines, baseDefines) {
 				}
 				//!steal-remove-start
 				else if (typeof result !== 'undefined') {
-					canLogDev.error("A Constructor or string must be used with shorthand definitions: https://canjs.com/doc/can-define.types.propDefinition.html#String");
+					canLogDev.error(prop + " on " + this.constructor.shortName + " does not match a supported propDefinition. See: https://canjs.com/doc/can-define.types.propDefinition.html");
```

Comment:
@bmomberger-bitovi are there times when `shortName` does not exist?

Reply:
DefineMaps/DefineLists have default shortNames, but a regular object that is defined wouldn't have them.